docs/content/docs/connectors/table/jdbc.md

```diff
@@ -458,6 +464,38 @@ SELECT * FROM mydb.`custom_schema.test_table2`;
 SELECT * FROM `custom_schema.test_table2`;
 ```
 
+### Jdbc Catalog for MySQL
+
+#### MySQL Metaspace Mapping
```

Comment:
Is this entire section really useful? For Postgres it exists because the mapping isn't trivial, but for MySQL it behaves exactly as expected. This is a lot of text for saying just that, and e.g. being able to use `database.table_name` or `table_name` is not specific to the MySQLCatalog, but just how Flink SQL works.

Reply:
Thank you @Airblader so much for comments. 
I just want to use this section to describe briefly that MySQL is almost smooth when mapping to the flink  catalog.
And please let me know what's your opinion about optimizing the section. 
Thank you.